addon/package.json

```diff
@@ -55,11 +59,28 @@
     "eslint-plugin-qunit": "^7.2.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.5.1",
-    "rollup": "^3.10.0"
+    "release-it": "^15.6.0",
+    "rollup": "^3.10.0",
+    "rollup-plugin-copy": "^3.4.0"
   },
   "engines": {
     "node": "14.* || >= 16"
   },
+  "release-it": {
```

Comment:
moved from the monorepo-root.

release-it doesn't need to support monorepos, because this one workspace is the only thing published to npm